src/interfaces.ts

```diff
@@ -5,6 +5,7 @@ export interface FormInputData {
   birthdate?: string;
   ageGate?: boolean;
   email: string;
+  preferredLanguage?: string;
```

Comment:
Why did you decide to keep the preferredLanguage as optional here? It seems to me that there will always be a value assigned?

Reply:
The value is empty for English since that's implicit. So it's only really needed for any other language and the default is "en".